src/fe-gtk/setup.c

```diff
@@ -556,14 +556,14 @@ static const setting advanced_settings[] =
 	{ST_HEADER,	N_("Auto Copy Behavior"),0,0,0},
 	{ST_TOGGLE, N_("Automatically copy selected text"), P_OFFINTNL(hex_text_autocopy_text),
 					N_("Copy selected text to clipboard when left mouse button is released. "
-						"Otherwise, CONTROL-SHIFT-C will copy the "
+						"Otherwise, Ctrl+Shift+C will copy the "
```

Comment:
Ok, any particular reason? I just followed this: https://developer.gnome.org/hig/stable/keyboard-input.html.en